### ISSUE TITLE
Trouble using meteor emails inside another package

### ISSUE THREAD

**Issue by mcoenca** [lookback/meteor-emails#28]
I think it is linked to the fact that you are not using Assets.getText(), and that i can't "push" files in the private/ directory from a package.js api.addFiles().

I get "could not get template MyTemplate Error 500" :( 

If somebody has a workaround, lmk ! Thanks

**Comment by mcoenca** [comment]
I am working on something, i have it working in dev. Will test in production and will submit pull request as soon as it is ready

**Comment by johanbrook** [comment]
Thanks!

**Comment by niranjans** [comment]
@mcoenca Did you manage to work around this? I'm stuck having that same "Could not find template.." message.

**Comment by mcoenca** [comment]
@niranjans 
Yes I have something here, https://github.com/mcoenca/meteor-emails/commit/ab4dea1d583bd494954146e8b70ca52de2d1e5aa , that works when you pass the underscore folder name of your package that requires lookback:emails (see .meteor/local/isopacks) 

`NotifsEmailTemplates.newQuestAnswer = {
  path: 'lib/newQuestAnswer.html',
  packageName: 'bridges_notifications',
  route:{
    path: '/...`

I have not merged or submitted pull request though, I am way behind the upstream master ^^ Will look into that

**Comment by mcoenca** [comment]
@niranjans Ok I've just checked. Compared to upstream, there have been breaking changes in code organisation around my commits, so I can't submit my pull request like this. You can either use my version (which lacks some of the newest features, and thus you need to use the old documentation on my branch), or try to look into my commit and adapt the concepts into your own fork from the new version. I don't have time right now to look at how to make the changes fit into the current upstream master Sorry !! :(

**Comment by niranjans** [comment]
@mcoenca no problem. I can look into your version and it would be a good start. I appreciate your help! :)

**Comment by adamgins** [comment]
HI, looking to leverage this too so I can use my email code/HTML into two apps. Just wanted to check if it had got any further/closer to release, please?

**Comment by johanbrook** [comment]
Is this still due to packages can't use `Assets.getText()` to get hold of stuff in the `private` directory?

**Comment by mcoenca** [comment]
Not sure anymore if my comment was right... you're reading files with fs anyway. I am looking if I can submit a valid PR tonight

**Comment by mcoenca** [comment]
Ok i almost have it, unsure about it working in all cases, needs testing, but less had changed than i thought so it should work. I've seen Meteor docs have been updated regarding Assets.getText() and package access, but as you are not using it to read files, I didn't want to break cohesion so I kept my old approach. #51
